[2021-12-27 12:19:11,130] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-27T12:19:08.248096+00:00 [queued]>
[2021-12-27 12:19:11,168] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-27T12:19:08.248096+00:00 [queued]>
[2021-12-27 12:19:11,185] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-27 12:19:11,195] {taskinstance.py:1242} INFO - Starting attempt 1 of 3
[2021-12-27 12:19:11,196] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-27 12:19:11,269] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): check_imageFile> on 2021-12-27 12:19:08.248096+00:00
[2021-12-27 12:19:11,311] {standard_task_runner.py:52} INFO - Started process 3698 to run task
[2021-12-27 12:19:11,329] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'check_imageFile', 'manual__2021-12-27T12:19:08.248096+00:00', '--job-id', '1604', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpz8f2lhd_', '--error-file', '/tmp/tmp10cx9adh']
[2021-12-27 12:19:11,338] {standard_task_runner.py:77} INFO - Job 1604: Subtask check_imageFile
[2021-12-27 12:19:11,728] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.check_imageFile manual__2021-12-27T12:19:08.248096+00:00 [running]> on host 218df5ddc363
[2021-12-27 12:19:12,143] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=check_imageFile
AIRFLOW_CTX_EXECUTION_DATE=2021-12-27T12:19:08.248096+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-27T12:19:08.248096+00:00
[2021-12-27 12:19:12,202] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-27 12:19:12,250] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sha256sum /opt/***/official_data/Image.csv']
[2021-12-27 12:19:12,298] {subprocess.py:85} INFO - Output:
[2021-12-27 12:19:12,344] {subprocess.py:89} INFO - e891870a24e5a22c9520c5d417af923366083a3617288cafa3cc18e3fed77b11  /opt/***/official_data/Image.csv
[2021-12-27 12:19:12,345] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-27 12:19:12,479] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=check_imageFile, execution_date=20211227T121908, start_date=20211227T121911, end_date=20211227T121912
[2021-12-27 12:19:12,718] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-27 12:19:12,952] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
